pkg/scalers/gcp_pub_sub_scaler.go

```diff
@@ -167,10 +170,18 @@ func (s *pubsubScaler) GetSubscriptionSize(ctx context.Context) (int64, error) {
 		}
 		s.client = client
 	}
+	var subscriptionID string
```

Comment:
Will it be better if this logic will be separated into new private method?

Reply:
It makes sense. I'll change that!

Reply:
Done!